### ISSUE TITLE
Github App Events forward unreliable since dependency upgrade

### ISSUE THREAD

**Issue by kruegener** [probot/smee.io#28]
I was developing a GitHub App and used smee.io to forward the API calls to localhost.
This worked perfectly for the last week including yesterday. Today upon starting work only about 25% of the calls we're properly registered on the smee.io page and forwarded.
I tried multiple different smee channels and different repos.

GitHub seems to not be the issue, as for the same repos all events were properly registered if I used my own server as endpoint instead of smee.

I presume this has to do with some of these updated dependencies from Pull Requests #23 -> #27.

**Comment by stefankip** [comment]
Yes we're also having issues, same report I guess: https://github.com/probot/smee-client/issues/132#issuecomment-525310194

**Comment by praveendvd** [comment]
@stefankip @kruegener  I don't think its issue with the smee-client but the smee server. Because i was using release version 1.1.0, i haven't used the latest code from the repo

**Comment by kruegener** [comment]
Sorry, but for me the service is still unusable at this time. Same unreliable behavior.

**Comment by stefankip** [comment]
Yep same for me.

**Comment by sssandyad** [comment]
up @JasonEtco

**Comment by bhumijgupta** [comment]
The issue still seems to persist for me. I am unable to receive any webhook event.

**Comment by JasonEtco** [comment]
@bhumijgupta I don't believe this is related to your issues, the Smee service has been healthy for the last week with no problems. If you'd like to open an issue and share some of your setup we can take a look!

**Comment by levimatheri** [comment]
Has been working perfectly for me since the traffic issues were addressed. Thank you @JasonEtco and team for all you do!
